modules/fedimint-ln-common/src/config.rs

```diff
@@ -20,18 +20,18 @@ pub struct LightningConfig {
     pub consensus: LightningConfigConsensus,
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize, Encodable)]
+#[derive(Debug, Clone, Serialize, Deserialize, Encodable, Decodable)]
 pub struct LightningConfigConsensus {
     /// The threshold public keys for encrypting the LN preimage
-    pub threshold_pub_keys: threshold_crypto::PublicKeySet,
+    pub threshold_pub_keys: SerdeEncodable<threshold_crypto::PublicKeySet>,
```

Comment:
I'm pretty sure that if `Encodable` was possible to impl, then we can also impl `Decodable`

Reply:
I noticed there were a few more places where `SerdeEncodable` was used. I removed all of them and implemented en/decoding for the types directly in 60767d2da7d710a16f6c9f65517bc3b8bbcf4c8d.